helpers/api.helper.ts

```diff
@@ -63,6 +69,134 @@ class ApiHelper {
         return user
     }
 
+    async createUnit(accessUserToken: string, unitName: string) {
+        const modelName = faker.lorem.word();
+        const firstName = faker.person.firstName();
+        const lastName = faker.person.lastName();
+        const description = faker.lorem.sentence();
+        const feature = faker.lorem.sentence();
+        const phoneNumber = `+38099${faker.string.numeric(7)}`;
+        const price = faker.number.int({min: 1000, max: 10000});
+
+        const response = await this.request
+              .post('https://dev.rentzila.com.ua/api/units/', {
+                headers: {
```

Comment:
You use the same header data almost every time. Therefore, create a variable for them and reuse it

Reply:
Ok.